vector/src/main/res/drawable/ic_alert_mark.xml

```diff
@@ -0,0 +1,4 @@
+<vector android:height="16dp" android:viewportHeight="286.054"
+    android:viewportWidth="286.054" android:width="16dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#E2574C" android:pathData="M143.027,0C64.04,0 0,64.04 0,143.027c0,78.996 64.04,143.027 143.027,143.027c78.996,0 143.027,-64.022 143.027,-143.027C286.054,64.04 222.022,0 143.027,0zM143.027,259.236c-64.183,0 -116.209,-52.026 -116.209,-116.209S78.844,26.818 143.027,26.818s116.209,52.026 116.209,116.209S207.21,259.236 143.027,259.236zM143.036,62.726c-10.244,0 -17.995,5.346 -17.995,13.981v79.201c0,8.644 7.75,13.972 17.995,13.972c9.994,0 17.995,-5.551 17.995,-13.972V76.707C161.03,68.277 153.03,62.726 143.036,62.726zM143.036,187.723c-9.842,0 -17.852,8.01 -17.852,17.86c0,9.833 8.01,17.843 17.852,17.843s17.843,-8.01 17.843,-17.843C160.878,195.732 152.878,187.723 143.036,187.723z"/>
```

Comment:
Can you confirm that vector work well on Android API 16?

Reply:
vector drawables are backward compatible, now . Just in build.gradle
have to make 
`defaultConfig{
     vectorDrawables.useSupportLibrary = true 
}`

Reply:
aside from `defaultConfig{ vectorDrawables.useSupportLibrary = true }`. one more line was supposed to be added 
 `AppCompatDelegate.setCompatVectorFromResourcesEnabled(true)` in the actvity or fragment

after adding these. It works fine. I have tested on API 16. 

<img src="https://user-images.githubusercontent.com/24780524/53187967-4ffcba80-362a-11e9-8cf8-59da13b7e719.png" width=360>

Reply:
ok